spec/lex.dd

```diff
@@ -355,7 +355,12 @@ $(GNAME TokenString):
 $(H3 $(LNAME2 wysiwyg, Wysiwyg Strings))
 
         $(P
-        Wysiwyg ("what you see is what you get") quoted strings are enclosed by `r"` and `"`.
+        Wysiwyg ("what you see is what you get") quoted strings can be defined
+        using either of two syntaxes.
+        )
+        
+        $(P
+        They can be enclosed by `r"` and `"`.
```

Comment:
enclosed between ?
And the following might be redundant, but what do you think of:
```suggestion
        In the first form, they are enclosed between `r"` and `"`.
```
?